.gitignore

```diff
@@ -1,3 +1,12 @@
 *.pyc
 .DS_Store
-config
\ No newline at end of file
+config
+/data
```

Comment:
I added this line because I don't want to save my data accidentally in Git.
However, I'm not sure to ignore data already committed in Git, so I added the following lines.